Library/Formula/orientdb.rb

```diff
@@ -3,9 +3,9 @@
 class Orientdb < Formula
   desc "Graph database"
   homepage 'http://www.orientdb.org/index.htm'
-  url 'http://www.orientechnologies.com/download.php?email=unknown@unknown.com&file=orientdb-community-2.0.11.tar.gz&os=mac'
-  version '2.0.11'
-  sha1 '812df4b971c3afc7bc9760b7de9bcc6680c060f0'
+  url 'http://www.orientechnologies.com/download.php?email=unknown@unknown.com&file=orientdb-community-2.0.12.tar.gz&os=mac'
```

Comment:
I'd suggest this secure `url` that also follows permanent redirect and omits unnecessary parameters:
`url "https://orientdb.com/download.php?file=orientdb-community-2.0.12.tar.gz"`